docs/docs/features/power-shaping.md

```diff
@@ -0,0 +1,21 @@
+# Power Shaping
+
+To give consumer chains more flexibility in choosing their validator set, Interchain Security v2
+introduces several "power shaping" mechanisms.
+
+These are:
+1) **Capping the size of the validator set**: The consumer chain can specify a maximum number of validators it
+wants to have in its validator set. This can be used to limit the number of validators in the set, which can
+be useful for chains that want to have a smaller validator set for faster blocks or lower overhead.
+*Note*: This is only applicable to Opt In chains (chains with Top N = 0).
+1) **Capping the fraction of power any single validator can have**: The consumer chain can specify a maximum fraction
+of the total voting power that any single validator in its validator set should have.
+This is a security measure with the intention of making it harder for a single large validator to take over a consumer chain. This mitigates the risk of an Opt In chain with only a few validators being dominated by a validator with a large amount of voting power opting in.
+For example, setting this fraction to e.g. 20% would mean that no single validator can have more than 20% of the total voting power,
```

Comment:
```suggestion
For example, setting this fraction to e.g. 33% would mean that no single validator can have more than 33% of the total voting power,
```
Because later you say "no single validator who would stop the chain by going offline" I believe it would make more sense to use 33% instead of 20% here. Because a single validator with 32% of voting power could still go down and the chain would still not halt.